src/deep-root-vanilla/frontend/bootstrap.js

```diff
@@ -12,4 +12,4 @@
   framework.Kernel.bootstrap(function () {
     framework.Kernel.get('deep_frontend_bootstrap_vector').forEach(importScript);
   });
-})(DeepFramework);
```

Comment:
Please undo to inner wrap, there are no sense to change. For more details please refer to: http://eslint.org/docs/rules/wrap-iife